package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.5.14",
   "description": "Zones for JavaScript",
   "main": "build/lib/zone.js",
-  "typings": "build/lib/core.d.ts",
```

Comment:
Don't remove this line - this is an output of the package. It allows users to
`import {Zone} from 'zone.js';`
and the typescript type-checker can find it.